config/config.php.dist

```diff
@@ -66,9 +67,10 @@ $config = [
      * When specified as a relative path, this is relative to the SimpleSAMLphp
      * root directory.
      */
+    'cachedir' => '/var/cache/simplesamlphp',
     //'loggingdir' => '/var/log/',
     //'datadir' => '/var/data/',
-    'tempdir' => '/tmp/simplesaml',
+    'tempdir' => '/tmp/simplesamlphp',
```

Comment:
Begs the question whether this is still used - will file a separate issue to discuss that.